react/Button/index.jsx

```diff
@@ -25,9 +25,9 @@ export const Button = props => {
       onClick={onClick}
     >
       <span>
-        {icon && <Icon icon={icon} />}
+        {typeof icon === 'string' ? <Icon icon={icon} /> : icon}
```

Comment:
It could be a breaking change here since we were able to pass the svg symbol as icon prop here, couldn't it?

Reply:
?

If the icon property is a string, we pass it to `<Icon />` otherwise we render the variable so I do not think it is a breaking change.

Reply:
Except that the `<Icon />` [can also take a SVG Symbol object](https://github.com/cozy/cozy-ui/blob/master/react/Icon/index.jsx#L10). This object is provided by the SVG loader when you import a SVG icon in the code. Before that, we were able to just pass this object to the Button to display using `<Icon />`, now we can't. I don't think we used a lot this way but it was just a warning about a potential breaking change.

Reply:
Good catch : https://github.com/cozy/cozy-ui/pull/377/commits/0ab5217a8618ff57b02dc37e39cbd3d4afa3600c . Is it ok ?